components/GraphDiagram/GraphDiagram.tsx

```diff
@@ -367,11 +365,8 @@ export function updateSelection(
   }
 
   // Set selection classes for edge elements
-  for (const titleEl of [...$<SVGElement>('svg .edge')]) {
-    const edgeTitle = $(titleEl, '.edge title')?.textContent ?? '';
-    const edge = $.up<SVGPathElement>(titleEl, '.edge');
-    if (!edge) continue;
-
+  for (const edge of $$('svg g.edge')) {
+    const edgeTitle = $('title', edge)?.textContent ?? '';
```

Comment:
Also fixed a bug here, it was not a `path` element so my `path.edge` selector was failing.


<table>
<tr>
	<th>Regression
	<th>Fixed
<tr>
	<td><img width="626" alt="Screenshot" src="https://github.com/user-attachments/assets/392184eb-73e6-4672-99a2-1795cdc094bf">
	<td><img width="626" alt="Screenshot 1" src="https://github.com/user-attachments/assets/40091278-99ca-4e62-98da-b867b3296421">
</table>







I'm not sure why it was selecting `.edge`, then `.edge title`, then using `$.up` to find `.edge`. I dropped that part and just used the first `.edge`